scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -227,6 +274,52 @@ task CalculateTargetCoverage {
     #}
 }
 
+# Add new columns to an existing target table with various targets
+task AnnotateTargets {
+    String entity_id
+    File padded_target_bed
+    File jar_file
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    Boolean disable_gc
+    Int mem
+
+    command {
+        if [ ${disable_gc} = false ]; \
+          then java -Xmx${mem}g -jar ${jar_file} AnnotateTargets --targets ${padded_target_bed} --reference ${ref_fasta} --output ${entity_id}.annotated.tsv; \
+          else touch ${entity_id}.annotated.tsv; \
+        fi
+    }
+
+    output {
+        File annotated_targets = "${entity_id}.annotated.tsv"
+    }
+}
+
+# Correct coverage for sample-specific GC bias effects
+task CorrectGCBias {
+    String entity_id
+    File coverage_file
+    File padded_target_bed
+    File annotated_targets
+    File jar_file
+    Boolean disable_gc
+    Int mem
+
+    command {
+        if [ ${disable_gc} = false ]; \
+          then java -Xmx${mem}g -jar ${jar_file} CorrectGCBias --input ${coverage_file} \
+           --output ${entity_id}.coverage.tsv --targets ${padded_target_bed} --targetAnnotations ${annotated_targets}; \
```

Comment:
I believe that instead of passing `--targets ${padded_target_bed} --targetAnnotations ${annotated_targets}` you may simply do `--targets ${annotated_targets}`.  See the `CorrectGCBiasIntegrationTest` for an example, and of course try it in case I'm wrong!  This is a consequence of the fact that there is no separate class `AnnotatedTarget extends Target`; rather `Target` itself has an `annotations` field.